modules/ob-installing-buildah-strategy.adoc

```diff
@@ -6,18 +6,19 @@
 [id="installing-buildah-strategy_{context}"]
 = Installing buildah strategy
 
-You can install the `buildah` cluster build strategy, which does not require a daemon to run and is available for unprivileged users. By using the `buildah-strategy-managed-push` resource for installing `buildah`, you can push the image to the target repository without any further processing.
-
+You can install the `buildah` strategy at cluster level. You can share the `buildah` strategy across different namespaces within your cluster by installing it at cluster level.
 
 .Prerequisites
 
 * You have installed the `oc` CLI.
+* You have installed the {builds-operator}.
+* You have created a `ShipwrightBuild` resource.
 
 .Procedure
 
 * To install the `buildah` strategy, run the following command:
 +
 [source,terminal]
 ----
-$ oc apply -f https://raw.githubusercontent.com/shipwright-io/build/main/samples/buildstrategy/buildah/buildstrategy_buildah_strategy_managed_push_cr.yaml
+$ oc apply -f https://raw.githubusercontent.com/redhat-developer/openshift-builds-catalog/main/clusterBuildStrategy/buildah/buildah.yaml
```

Comment:
(Here and the other similar references to GitHub URLs)

Technically per our [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites), we are not supposed to include links or source content directly from GitHub without approval from our Support org. Maybe the previous URL being replaced here was approved at some point, but I couldn't verify in the PR or Jira if the new one was. At any rate, I checked w/ kalexander on this one and we're OK to move it along for now (since this is replacing an existing GitHub link anyway), but please consider for future changes. Thank you!

Reply:
@adellape, Thank you for sharing this information. In a future release, plans are to move this entire section of `Installing build strategies` because users need not to install the strategies manually. The Builds Operator would automatically install all these strategies.

Reply:
Hey @adellape, the DevTools teams have exceptions to use links from our support team to examples or repos that are maintained by the OpenShift and Developer org in GH.